CSC8503/SuspicionSystem/GlobalSuspicionMetre.cpp

```diff
@@ -0,0 +1,63 @@
+#include "GlobalSuspicionMetre.h"
+#include <algorithm>
+
+void GlobalSuspicionMetre::AddInstantGlobalSusCause(instantGlobalSusCause inCause)
+{
+    ChangePlayerGlobalSusMetre(mInstantCauseSusSeverityMap[inCause]);
+}
+
+bool GlobalSuspicionMetre::AddActiveGlobalSusCause(activeGlobalSusCause inCause)
+{
+    auto foundCause = std::find(mActiveGlobalSusCauseVector.begin(), mActiveGlobalSusCauseVector.end(), inCause);
+
+    //If the foundCause is not already in the mActiveGlobalSusCauseVector vector of that player
```

Comment:
comment not needed - I think it's pretty clear as is that you're using an iterator, so reaching .end() means it was not found